enclave-runtime/src/lib.rs

```diff
@@ -609,6 +591,61 @@ where
 
 	LightClientSeal::seal(validator)?;
 
+	drop(light_client_lock);
+	drop(side_chain_lock);
```

Comment:
Inferring from the article you mentioned above, I think it would fine to use `_`-prefixed variables without the explicit drop. It doesn't hurt to be sure, though.